src/scss/components/local-footer/_local-footer.scss

```diff
@@ -239,15 +241,19 @@
 nav[aria-label^='footer'] {
   a,
   a.su-link--external,
-  a.su-link {
+  a.su-link,
+  .su-local-footer__action-links {
     color: $chemh-color-digital-blue;
     text-decoration: underline;
 
     &:hover,
     &:focus {
```

Comment:
Can you adjust this for active as well? Other changes look good!
![image](https://user-images.githubusercontent.com/49299083/236524078-6f76ea21-060b-45f3-881d-22d138923231.png)